packages/client/hmi-client/src/components/widgets/tera-tab-group.vue

```diff
@@ -92,8 +114,10 @@ header {
 	display: none;
 	visibility: hidden;
 }
+
 .p-button {
-	flex-shrink: 0; /* Avoid the highlight to be squished */
+	flex-shrink: 0;
+	/* Avoid the highlight to be squished */
```

Comment:
If it can't be in one line, then the comment should be above

```suggestion
	/* Avoid the highlight to be squished */
	flex-shrink: 0;
```